generated/nidcpower/nidcpower.proto

```diff
@@ -1295,7 +1412,7 @@ message ExportAttributeConfigurationBufferRequest {
 
 message ExportAttributeConfigurationBufferResponse {
   int32 status = 1;
-  repeated fixed64 configuration = 2;
+  bytes configuration = 2;
```

Comment:
I think these are compatibility breaking. I'm not 100% sure on what the behavior is. It's possible that it works, especially if these are only used in an `export`/`import` roundtrip (with multiples of 4 bytes?).

Assuming it's breaking, it would be OK to just flag the change if that's OK with Harsha and someone from DCPower.

Alternatively, we could:
1. Add the new version as a new field and deprecate the old one.
   * You could reuse the field name but change the index. 
1. Make a `oneof` and handle both.

Reply:
Marked as resolved. Removed `nidcpower` from this PR since Ryan will be updating it.